common/djangoapps/student/views/management.py

```diff
@@ -1185,6 +1186,7 @@ def password_reset_confirm_wrapper(request, uidb36=None, token=None):
         password = request.POST['new_password1']
 
         try:
+            password = unicode_check(password)
             validate_password(password, user=user)
```

Comment:
I'd like to talk to you more about this in light of the unicode work Cali is doing and if you think it still makes sense to separate this into its own function. I agree with all your logic and am happy to consolidate it, but I just want to make sure it is worthwhile work.

Reply:
I think so? I think it would actually help that eventual PR, rather than hurt it, as there will be one place to insert normalization logic.

Imagine Cali (or whoever) coming to this code afterward. If they have three places to edit, that's more effort, makes those code locations messier, and will make the implementor wonder if there are other places that they are missing.